wolfssl/wolfcrypt/ecc.h

```diff
@@ -422,6 +422,7 @@ WOLFSSL_ABI WOLFSSL_API void wc_ecc_key_free(ecc_key*);
 
 /* ECC predefined curve sets  */
 extern const ecc_set_type ecc_sets[];
+extern const size_t ecc_sets_count;
```

Comment:
ecc_sets has ECC_CURVE_INVALID as last entry.
Why do you need ecc_sets_count?

Reply:
I used `ecc_sets_count` in `wolfSSL_EC_get_builtin_curves` so that it doesn't have to be computed at each function call.